sal_interfaces/SALSubsystems.xml

```diff
@@ -408,16 +408,16 @@
   <Generics>yes</Generics>
   <ActiveDevelopers>Eric Coughlin</ActiveDevelopers>
   <Github>https://github.com/lsst-ts/ts_salobjATHexapod</Github>
-  <JenkinsTestResults>Not Available</JenkinsTestResults>
+  <JenkinsTestResults>https://tssw-ci.lsst.org/job/LSST_Telescope-and-Site/job/ts_athexapod/</JenkinsTestResults>
   <RubinObsContact>Patrick Ingraham</RubinObsContact>
   <CSCDocs>NA</CSCDocs>
   <ProductOwner>Patrick Ingraham</ProductOwner>
   <RelatedDocuments>There is one, ask patrick</RelatedDocuments>
   <SoftwareLanguage>Python</SoftwareLanguage>
   <RuntimeLanguages>IDL</RuntimeLanguages>
   <VendorContact>Physik Instrumente (PI)</VendorContact>
-  <Simulator>Yes</Simulator>
-  <Configuration>Yes</Configuration>
+  <Simulator>https://repo-nexus.lsst.org/nexus/#browse/browse:ts-dockerhub:v2%2Fhexapod_simulator%2Fmanifests (private)</Simulator>
```

Comment:
This doesn't look like a code repository. Does it have one?

Reply:
its the container with the simulator... it is private code.. it's the only "link" I could think off.

Reply:
OK. In this we'll use what we have.